dev/com.ibm.ws.concurrent.persistent_fat/test-applications/schedtest/src/web/SchedulerFATServlet.java

```diff
@@ -3830,9 +3853,13 @@ public Integer call() throws Exception {
                     if (status.hasResult())
                         throw new Exception("Task status initial snapshot should not have a result. " + status);
 
-                    status = pollForResult(status.getTaskId(), 3);
+                    long taskId = status.getTaskId();
+                    for (long start = System.nanoTime(); System.nanoTime() - start < TIMEOUT_NS
+                            && (status = scheduler.getStatus(taskId)) != null
+                            && !(status.hasResult() && Integer.valueOf(1).equals(status.getResult())); )
```

Comment:
Should Integer.valueOf(1) here be Integer.valueOf(3) instead?

Reply:
The task is only scheduled to run once, which returns a value of 1.  Waiting for 3, which was in the previous code, was erroneous, and I'm fixing it here.